kronolith/lib/Ajax/Application/Handler.php

```diff
@@ -352,10 +352,15 @@ public function saveEvent()
         }
 
         if (($result !== true) && $this->vars->sendupdates) {
-            $type = $event->status == Kronolith::STATUS_CANCELLED
-                ? Kronolith::ITIP_CANCEL
-                : Kronolith::ITIP_REQUEST;
-            Kronolith::sendITipNotifications($event, $notification, $type);
+            if ($this->vars->attendance) {
+                Kronolith::sendITipNotifications($event, $notification, Kronolith::ITIP_REPLY);
+            }
+            if (!$event->organizer || Kronolith::isUserEmail($this->creator, $event->organizer)) {
+                $type = $event->status == Kronolith::STATUS_CANCELLED
+                    ? Kronolith::ITIP_CANCEL
+                    : Kronolith::ITIP_REQUEST;
+                Kronolith::sendITipNotifications($event, $notification, $type);
+            }
```

Comment:
In addition to these checks (or possibly even instead of these), we need to check this on the client side and prevent the dialog on the dynamic view that asks if we want to send updates. In other words, !event.oy, then don't show the dialog and set sendupdates to false.